src/index.html

```diff
@@ -1,18 +1,78 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="card"
```

Comment:
it's bad idea to give the class to HTML tag, better to create separate block(card tag inside the body) so you can easily reuse it